### DISCUSSION TITLE
No Master Branch available?

### DISCUSSION THREAD

**Discussion by Memati53** [mupen64plus-ae/mupen64plus-ae#1148] [General]
Hello,

I have read about "M64Plus FZ" being the same as "Mupen64Plus-AE" but without any ads. 
Also that one should get the master branch version but I can't download it due to it being expired (5 months old). 

I would like to know if "mupen64plus-ae-improved_deadzone-e8724b1" should be downloaded or rather not?
What would be the alternative for an ad-free version of M64Plus FZ.

Thanks in advance

**Comment by fzurita** [comment]
Yeah, you can download it. There may be some issues though with the touchscreen graphics.

**Reply by Memati53** [reply]
Thanks for your help. Are there also such issues with the "M64Plus FZ Pro Emulator" on Google Play by any chance?

**Comment by fzurita** [comment]
There are not.
